data/hai/unfettered 0 prologue.txt

```diff
@@ -224,7 +234,12 @@ mission "Unfettered Jump Drive 2"
 				`	"Sorry, I need this jump drive right now, but I will come back later and sell it to you."`
 					defer
 				`	"Yes, I will sell you my jump drive."`
-			`	Once again, they swap your jump drive for a hyperdrive, and pay you far more than you could sell a jump drive for anywhere else. "We will tell the other true Hai to offer you jobs in our job boards," they tell you, "and remember that we need still more jump drives."`
+			branch cargo
+				has "outfit (cargo): Jump Drive"
+			action
+				outfit "Hyperdrive" 1
+			label cargo
+			`	Once again, they offer to swap your jump drive for a hyperdrive, and pay you far more than you could sell a jump drive for anywhere else. "We will tell the other true Hai to offer you jobs in our job boards," they tell you, "and remember that we need still more jump drives."`
```

Comment:
```suggestion
			`	Once again, they offer to swap your jump drive for a hyperdrive, and pay you far more than you could sell a jump drive for anywhere else. "We will tell the other true Hai to offer you jobs in our job boards," they tell you, "and remember that we need still more jump drives."`
```
I think it's a little odd here that you change it to say "they offer to" here, but there is no given option for the player to respond. Above, there isn't really a choice, but there is still a "choice" displayed to the player that they don't need to give them the HD. Here it implies they give a choice but then nothing like that happens; I think it would be better to just keep the original line.